private-templates-service/client/src/components/Common/Tags/Tags.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import {
+  Tag,
+  TagCloseIcon,
+  TagText,
+  AddTagWrapper,
+  AddTagInput,
+  TagAddIcon,
+  TagSubmitButton,
+  TagSubmitIcon,
+} from './styled';
+
+interface Props {
+  tags?: string[];
+  allowAddTag?: boolean;
+}
+
+interface State {
+  isAdding: boolean;
+  newTagName: string;
+}
+
+class Tags extends React.Component<Props, State>  {
```

Comment:
Is this part of the stage pipeline bug fix?

Reply:
No, another PR was pushed while I was waiting for this one to be reviewed so the changes got added (since we're merging dev into stage).